maven/plugins/com.liferay.ide.maven.core/src/com/liferay/ide/maven/core/MavenModuleProjectImporter.java

```diff
@@ -70,4 +114,182 @@ public IStatus canImport( String location )
         return projects;
     }
 
+    @Override
+    public void updateBuildFile( String location ) throws Exception
```

Comment:
This should not be done on import.  Rather, this type of change should be added to Code Upgrade Tool as a separate step.  We can't modify user's POM files on import.  That would be unexpected behavior.

We need this to be a part of `find breaking changes` or rather maybe a new `update POMs` page.  This piece will be hard to do since the `project.ui` can't depend on maven.  So likely right now putting it in `find breaking changes` may be the easiest.